.agents/skills/caveman-compress/scripts/detect.py

```
@@ -0,0 +1,121 @@
+            text = filepath.read_text(errors="ignore")
+        except (OSError, PermissionError):
+            return "unknown"
+
+        lines = text.splitlines()[:50]
+
+        if _is_json_content(text[:10000]):
+            return "config"
+        if _is_yaml_content(lines):
+            return "config"
+
+        code_lines = sum(1 for l in lines if l.strip() and _is_code_line(l))
```

Ruff (0.15.17)

[error] Ambiguous variable name: `l`

(E741)